### PR TITLE
feat: hide help me choose feat when loans are fewer than 9 in categor…

### PR DESCRIPTION
**Pull request by roger-in-kiva**
Help me choose feature hidden in category page when the number of loans is fewer than 9

### CODE REVIEW

File: src/pages/Lend/LoanChannelCategoryControl.vue [hunk 1]
```diff
@@ -421,6 +421,10 @@ export default {
 		},
 		hasOnePageOfLoans() {
 			return this.totalCount <= this.limit;
+		},
+		showHelpMeChooseFeat() {
+			if (this.enableHelpmeChoose && this.allLoans.length > 8) return true;
+			return false;
```
Comment (mcstover) [inline src/pages/Lend/LoanChannelCategoryControl.vue:427]: Couldn't this just be `return this.enableHelpmeChoose && this.allLoans.length > 8;` without the if block and second return?

Comment (roger-in-kiva) [reply src/pages/Lend/LoanChannelCategoryControl.vue]: oooh yes 🤯
fixed

File: src/pages/Lend/LoanChannelCategoryControl.vue [hunk 2]
```diff
@@ -421,6 +421,9 @@ export default {
 		},
 		hasOnePageOfLoans() {
 			return this.totalCount <= this.limit;
+		},
+		showHelpMeChooseFeat() {
+			return this.enableHelpmeChoose && this.allLoans.length > 8;
```
Comment (dyersituations) [inline src/pages/Lend/LoanChannelCategoryControl.vue:426]: Should we be using `this.allLoans.length` or the `totalCount` computed prop?

Comment (roger-in-kiva) [reply src/pages/Lend/LoanChannelCategoryControl.vue:426]: Looks like `totalCount` is for all loans in the channel. `this.allLoans.length` is counting current loans in the page.

Logging this, I have a different number for livestoc for ex.
94 with `totalCount`
12 with `this.allLoans.length`

Comment (dyersituations) [reply src/pages/Lend/LoanChannelCategoryControl.vue:426]: Interesting that `totalCount` doesn't appear to be respecting filters. Thanks for investigating!

**Review by dyersituations — APPROVED** [review]

**Pull request merged**